### PR TITLE
Qrcode on windows terminal

### PR DESCRIPTION
**Pull request by Seven-Y-Q-Guo**
https://github.com/parvardegr/sharing/issues/20

After checking the issues of qrcode-terminal, seems windows os can't support small option to output the qrcode in native terminal.

In this PR just adding -w to ignore the small will resolve this issue.

Example:
```bash
sharing f:\demo\demo\pom.xml -w
```

Before:
image

After:
image

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -35,13 +36,19 @@ var getNetworkAddress = () => {
...
         .option("c", { alias: 'clipboard', describe: "Share Clipboard", type: "boolean", demandOption: false })
+        .option("w", { alias: 'windows', describe: "Is windows os", type: "boolean", demandOption: false })
         .help(true)
         .argv;
 
 
     let path = undefined;
     let fileName = undefined;
 
+    if (options.windows) {
```
Comment (parvardegr) [inline bin/index.js]: Hi @Seven-Y-Q-Guo 
another nice PR ^_^
I think we can remove the `-w` option and change this to something like this:
```
if (process.platform === 'win32') {
   ... Do stuff for windows
}
```
Thanks a lot

[process.platform](http://nodejs.org/api/process.html#process_process_platform)

Comment (Seven-Y-Q-Guo) [reply bin/index.js]: Yeal, I was also thought about this idea in my mind, but after confirming the issue will only happen on Windows via native terminal. If use others like powershell, git bash, etc. It'll be working fine.
Use this logic will make qrcode big for all on Windows than ever before, does it make sense?

Comment (parvardegr) [reply bin/index.js]: you're right. in that case, we need to make this option name and description a little more accurate.
```
.option("on-windows-native-terminal", { describe: "enable QR-Code support for windows native terminal", type: "boolean", demandOption: false })
```
the above line is just an example, you may come up with a better name and description. 

great regards

**Comment by Seven-Y-Q-Guo** [comment]
@parvardegr updated.

**Pull request merged**